lib/publiccloud/instance.pm

```diff
@@ -79,14 +79,16 @@ sub run_ssh_command {
     if ($args{timeout} == 0) {
```

Comment:
If you like, not part of your change, there's a typo in the sub description, two consecutive stops
`If the command retrieves not zero, a exception is thrown..` should be `If the command retrieves not zero, a exception is thrown.`

Reply:
Done.
`s/.././`
`s/a exception/an exception/`